packages/kbn-i18n/src/core/i18n.ts

```diff
@@ -216,7 +221,7 @@ export function translate(
  * Initializes the engine
  * @param newMessages
  */
-export function init(newMessages?: PlainMessages) {
+export function init(newMessages?: Translation) {
```

Comment:
__nit:__ `newMessages` ---> `newTranslation` (don't forget to update JSDoc as well)